app/main.py

```diff
@@ -1,6 +1,23 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+    customers: list[dict],
+    hall_number: int,
+    cleaner_name: str,
+    movie: str
+) -> None:
+    customers = [Customer(
+        customer["name"],
+        customer["food"]
+    )
+        for customer in customers]
```

Comment:
It would be better to format this list comprehension in another way to avoid unnecessary line breaking:
```suggestion
    customers = [Customer(customer["name"], customer["food"])
                 for customer in customers]
```
Prefer this way in similar situations in the future.

Reply:
Thanks for your advice. I'll try to do as you.